src/structures/Emoji.js

```diff
@@ -12,9 +12,9 @@ class Emoji extends Base {
     super(client);
     /**
      * Whether this emoji is animated
-     * @type {boolean}
+     * @type {?boolean}
      */
-    this.animated = emoji.animated;
+    this.animated = emoji.animated ?? null;
```

Comment:
The [typings](https://github.com/almostSouji/discord.js/blob/b80277e2785ce38df5d39b85494b34766892236f/typings/index.d.ts#L578) need to be updated for this